bash_completionsV2.go

```diff
@@ -220,7 +220,14 @@ __%[1]s_handle_standard_completion_case() {
 
     # Short circuit to optimize if we don't have descriptions
     if [[ "${completions[*]}" != *$tab* ]]; then
-        IFS=$'\n' read -ra COMPREPLY -d '' < <(compgen -W "${completions[*]}" -- "$cur")
+        compgen_words=$(printf "%%s\n" "${completions[@]}")
+        IFS=$'\n' read -ra COMPREPLY -d '' < <(IFS=$'\n' compgen -W "${compgen_words}" -- "$cur")
+
+        # If there is a single completion left, escape the completion
+        if [ ${#COMPREPLY[*]} -eq 1 ]; then
+            COMPREPLY[0]=$(printf %%q "${COMPREPLY[0]}")
```

Comment:
I think we can eliminate a subshell here.
```suggestion
            printf -v "COMPREPLY[0]" %%q "${COMPREPLY[0]}"
```

Reply:
Hmm, actually `printf -v` to an array index is a bash 4.1+ thing, so as/if older compatibility is needed, then scratch this and similar comments of mine elsewhere.

Reply:
Yeah, we try to keep it working with bash 3, as much as possible.